package.json

```diff
@@ -1,34 +1,12 @@
 {
-  "name": "@cinchy-co/angular-sdk",
-  "version": "4.4.0",
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/cinchy-co/angular-sdk"
-  },
-  "author": {
-    "name": "Cinchy",
-    "email": "opensource@cinchy.co"
-  },
-  "keywords": [
-    "angular",
-    "cinchy",
-    "api",
-    "data network",
-    "data collabortation"
-  ],
-  "license": "GPL-3.0-only",
-  "bugs": {
-    "url": "https://github.com/cinchy-co/angular-sdk/issues"
-  },
-  "peerDependencies": {
-    "@angular/core": "^6.0.0",
-    "zone.js": "^0.8.4"
-  },
+  "name": "angular-sdk",
+  "version": "12.0.0",
   "dependencies": {
-    "angular-oauth2-oidc": "^10.0.3",
-    "jsrsasign": "^8.0.24",
-    "tslib": "^2.0.0",
-    "iframe-resizer": "^3.6.1",
-    "rxjs-compat": "^6.2.0"
+    "@types/jsrsasign": "^10.5.1",
+    "angular-oauth2-oidc": "^13.0.1",
+    "iframe-resizer": "^4.3.2",
```

Comment:
Have we tested this new iframe-resizer version by embedding an applet within Cinchy?